scionlab/templates/scionlab/user_as_add.html

```diff
@@ -5,11 +5,16 @@
 {% block title %}Create User AS{% endblock %}
 
 {% block content %}
-<div class="form-box">
-  {% crispy form %}
-  <button type="submit" form="id_user_as_form" class="btn btn-primary savebtn">Create AS</button>
-</div>
+    <form action="{% url 'user_as_add' %}" id="id_user_as_form" method="post">
```

Comment:
I liked the `form-box` :)

Reply:
Were would you like to insert it back? Before it was wrapping the whole form, but now there are multiple sections divided with their own boxes.